### PR TITLE
Adding working .travis.yml example

### PR DESCRIPTION
**Pull request by Andrew5194**
Solves: [MH-13503]

This PR adds a working TravisCI example for `mcode-action-examples`.

https://github.com/ForAllSecure/mcode-action-examples/assets/20665775/fa5a699c-d23e-49af-80b1-9d4e42b846bf

[MH-13503]: https://forallsecure.atlassian.net/browse/MH-13503?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Andrew5194** [comment]
Seems like TravisCI can only upload artifacts to AWS S3, see https://docs.travis-ci.com/user/uploading-artifacts/. Wasn't sure if we wanted to include this in this tutorial workflow, but let me know if so.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+stages:
+  - build
+
+jobs:
+  include:
+    - stage: build
+      name: "Build Job"
+      script:
+        # Build and push Docker image to registry
+        - docker login ghcr.io -u $GITHUB_USERNAME -p $GITHUB_TOKEN
```
Comment (vlussenburg) [inline .travis.yml:13]: Why aren't we pushing to Mayhem's internal registry? Seems to be easier and avoid a bunch of secrets to maintain

Comment (Andrew5194) [reply .travis.yml:13]: I thought we weren't publicly showcasing the internal Mayhem Docker Registry for online deployments (i.e. app.mayhem.security). For example, in the past for `mayhem.forallsecure.com`, we deliberately hid the internal Docker registry stuff, see the following for comparison:
* https://beta.forallsecure.com/-/create/run/docker?registry=mayhem <-- enterprise deployment variant

```
{"environment":"gke_oss-000042_us-west1-a_mayhem","name":"Mayhem API","offline":false,"public":true,"version":"2.5.0"}
```

* https://mayhem.forallsecure.com/-/create/run/docker?slide=0&registry=mayhem <-- online deployment variant

```
{"environment":"gke_oss-000042_us-west1-a_mayhem","name":"Mayhem API","offline":false,"public":true,"version":"2.5.0"}
```

Comment (vlussenburg) [reply .travis.yml:13]: Ah. Fair :)

**Review by vlussenburg — APPROVED** [review]

**Pull request merged**